orchagent/p4orch/tests/mock_sai_udf.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
could you please remove the copyright? and also for any other files in the PR?

Reply:
Done